README.md

```diff
@@ -1,130 +1,181 @@
 
----
 
-## What you'll find here
 
-The Fission Webnative SDK offers tools for:
-- authenticating through a Fission **authentication lobby**  
-  (a lobby is where you can make a Fission account or link an account)
-- managing your web native **file system**  
-  (this is where a user's data lives)
-- tools for building DIDs and UCANs.
-- interacting with the users apps via the **platform APIs**
+# Getting started
 
 ```ts
-// ES6
-import * as wn from 'webnative'
+// ESM
+import * as wn from "webnative"
 
 // Browser/UMD build
 const wn = globalThis.webnative
 ```
 
-# Authentication
+## Creating a Program
+
+A Webnative program is an assembly that makes up a part of, or a whole distributed web application. It consists out of several components that can be customised. _We'll stick with the default components for now, which means we'll be using the Web Crypto API._
 
 ```ts
-const state = await wn.initialise({
-  permissions: {
-    // Will ask the user permission to store
-    // your apps data in `private/Apps/Nullsoft/Winamp`
-    app: {
-      name: "Winamp",
-      creator: "Nullsoft"
-    },
-
-    // Ask the user permission to additional filesystem paths
-    fs: {
-      private: [ wn.path.directory("Audio", "Music") ],
-      public: [ wn.path.directory("Audio", "Mixtapes") ]
-    }
+const program = await wn.program({
+  // Can also be a string, used as an identifier for caches.
+  // If you're developing multiple apps on the same localhost port,
+  // make sure these differ.
+  id: { creator: "Nullsoft", name: "Winamp" }
+
+}).catch(error => {
+  switch (error) {
+    case webnative.ProgramError.InsecureContext:
+      // Webnative requires HTTPS
+      break;
+    case webnative.ProgramError.UnsupportedBrowser:
+      break;
   }
 
-}).catch(err => {
-  switch (err) {
-    case wn.InitialisationError.InsecureContext:
-      // We need a secure context to do cryptography
-      // Usually this means we need HTTPS or localhost
+})
+```
 
-    case wn.InitialisationError.UnsupportedBrowser:
-      // Browser not supported.
-      // Example: Firefox private mode can't use indexedDB.
-  }
+That gives us a `Program` object, with this we can create a new user session or reuse the existing session. There's two distinctive ways to create a user session, either by using an authentication strategy or by using something we call "confidences". More on the latter in a bit, let's start with the default authentication strategy.
 
-})
+```ts
+let session
+
+// Do we have an existing session?
+if (program.session) {
+  session = program.session
+
+// If not, let's authenticate.
+// (a) new user, register a new Fission account
+} else if (registerNewUser) {
```

Comment:
How would a developer decide what to set `registerNewUser` to?

Reply:
It's decided by the user wether or not they have an account, eg. this screen in WAT:

<img width="393" alt="Screenshot 2022-11-18 at 11 23 59" src="https://user-images.githubusercontent.com/296665/202680102-4e4fc9ec-6eba-4a9f-9c9f-8ca8b6ad9e7c.png">

The wording could be better here yeah.
Maybe `userChoseToRegister`?

Reply:
Yeah, that wording would be better! 👌